heat/core/manipulations.py

```diff
@@ -25,24 +26,118 @@
     "hstack",
     "reshape",
     "resplit",
+    "row_stack",
     "shape",
     "sort",
     "squeeze",
+    "stack",
     "topk",
     "unique",
     "vstack",
 ]
 
 
+def column_stack(arrays):
+    """
+    Stack 1-D or 2-D ``DNDarray``s as columns into a 2-D ``DNDarray``.
+    If the input arrays are 1-D, they will be stacked as columns. If they are 2-D,
+    they will be concatenated along the second axis.
+
+    Parameters
+    ----------
+    arrays : Sequence[DNDarrays,...]
+
+    Raises
+    ------
+    ValueError
+        If arrays have more than 2 dimensions
+
+    Returns
+    -------
+    DNDarray
+
+    Note
+    ----
+    All ``DNDarray``s in the sequence must have the same number of rows.
+    All ``DNDarray``s must be split along the same axis! Note that distributed
+    1-D arrays (``split = 0``) by default will be transposed into distributed
+    column arrays with ``split == 1``.
+
+    Examples
+    --------
+    # 1-D tensors
+    >>> a = ht.array([1, 2, 3])
+    >>> b = ht.array([2, 3, 4])
+    >>> ht.column_stack((a, b))
+    tensor([[1, 2],
+        [2, 3],
+        [3, 4]])
+    # 1-D and 2-D tensors
+    >>> a = ht.array([1, 2, 3])
+    >>> b = ht.array([[2, 5], [3, 6], [4, 7]])
+    >>> c = ht.array([[7, 10], [8, 11], [9, 12]])
+    >>> ht.column_stack((a, b, c))
+    tensor([[ 1,  2,  5,  7, 10],
+            [ 2,  3,  6,  8, 11],
+            [ 3,  4,  7,  9, 12]])
+    # distributed DNDarrays, 3 processes
+    >>> a = ht.arange(10, split=0).reshape((5, 2))
+    >>> b = ht.arange(5, 20, split=0).reshape((5, 3))
+    >>> c = ht.arange(20, 40, split=0).reshape((5, 4))
+    >>> ht_column_stack((a, b, c))
+    [1,0] tensor([[ 0,  1,  5,  6,  7, 20, 21, 22, 23],
+    [1,0]         [ 2,  3,  8,  9, 10, 24, 25, 26, 27]], dtype=torch.int32)
+    [1,1] tensor([[ 4,  5, 11, 12, 13, 28, 29, 30, 31],
+    [1,1]         [ 6,  7, 14, 15, 16, 32, 33, 34, 35]], dtype=torch.int32)
+    [1,2] tensor([[ 8,  9, 17, 18, 19, 36, 37, 38, 39]], dtype=torch.int32)
+    # distributed 1-D and 2-D DNDarrays, 3 processes
```

Comment:
the comments here should have `>>>` in front of them. that way the examples will render correctly

Reply:
Done everywhere, thanks.